ext/php8/configuration.h

```diff
@@ -4,10 +4,38 @@
 #include <stdbool.h>
 
 #include "compatibility.h"
+#include "ddtrace_config.h"
 #include "ddtrace_string.h"
 #include "env_config.h"
 #include "integrations/integrations.h"
 
+// TODO Tie these into the X Macros below
+static inline bool get_dd_trace_debug(void) {
+    zval *val = zai_config_get_value(DDTRACE_CONFIG_DD_TRACE_DEBUG);
+    return Z_TYPE_P(val) == IS_TRUE;
+}
+
+static inline zai_string_view get_dd_service(void) {
+    zval *val = zai_config_get_value(DDTRACE_CONFIG_DD_SERVICE);
+    return (zai_string_view){.len = Z_STRLEN_P(val), .ptr = Z_STRVAL_P(val)};
```

Comment:
I see no real gain here in converting to a zai_string_view, especially as some options like DD_SERVICE will end just straight up in the meta array. I.e. us ending up cycling the datastructure back and forth. Just return the string.
Also this string_view may not be stored as is, as its lifetime is dependent on the config value (i.e. when the value is changed and nothing pointing to it, this will point to unrelated memory). Keep it simple and don't add new pitfalls here.

Reply:
Yes, these are very good points. Returning a `zend_string` on PHP 7+ would make sense to me.